src/Box/utils.ts

```diff
@@ -0,0 +1,35 @@
+import { css } from 'styled-components';
+import { getColorFromTheme } from '../shared/mixins';
+import { Theme } from '../shared/theme.types';
+import { BORDER_RADIUS_MAPPING, BOX_SHADOW_MAPPING } from './constants';
+import { StyledBoxProps } from './types';
+
+export const getBackgroundStyles = ({ $background, theme }: StyledBoxProps & { theme: Theme }) => {
+  if ($background) {
+    return css`
+      background-color: ${getColorFromTheme(theme, $background)};
```

Comment:
@david-gomey @solilokiam @amalv I'm not sure if `background` should be the name of the prop or `backgroundColor`, to correspond the css rule. What do you think?

Reply:
`backgroundColor` would be nice, in my opinion

Reply:
Agree, changed!